src/dotnet/APIView/APIViewWeb/Program.cs

```diff
@@ -17,14 +18,14 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
                 .ConfigureAppConfiguration((hostingContext, config) =>
                 {
-                    config.AddEnvironmentVariables(prefix: "APIVIEW_");                  
+                    config.AddEnvironmentVariables(prefix: "APIVIEW_");
                     IConfiguration settings = config.Build();
-                    string connectionString = settings.GetValue<string>("APPCONFIG");
+                    string endpoint = settings.GetValue<string>("AppConfigEndpoint");
                     // Load configuration from Azure App Configuration
                     config.AddAzureAppConfiguration(options =>
                     {
-                        options.Connect(connectionString).ConfigureKeyVault(kv => 
-                        { 
+                        options.Connect(new Uri(endpoint), new DefaultAzureCredential()).ConfigureKeyVault(kv =>
```

Comment:
hope we could switch to use entra-id for app config as well like we did for storage blob and cosmos